dll/appcompat/shims/layer/versionlie.c

```diff
@@ -230,6 +231,9 @@ BOOL WINAPI SHIM_OBJ_NAME(APIHook_GetVersionExW)(LPOSVERSIONINFOEXA lpOsVersionI
 #define VERSION_INFO    g_Win7RTM
 #include "versionlie.inl"
 
+#define SHIM_NS         Win7SP1VersionLie
```

Comment:
Does windows provide this shim?
If so, can you please add an apitest showing that this shim reports the correct values?

Reply:
>can you please add an apitest showing that this shim reports the correct values?

ok, i thought adding "run_test("Win7SP1VersionLie", &g_Win7SP1);" is enough

Reply:
Can you tell what version number Windows 8 and/or 10 claim when they are set to Windows 7 compatibility mode?

Reply:
I am seeing `WinSrv08SP1` but no `Win7SP1`